ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/frontend.hpp

```diff
@@ -64,7 +64,7 @@ namespace ngraph
             virtual InputModel::Ptr
                 loadFromStreams(const std::vector<std::istream*>& paths) const override;
 
-            virtual std::shared_ptr<Function> convert(InputModel::Ptr model) const override;
+            std::shared_ptr<Function> convert(InputModel::Ptr model) const override;
```

Comment:
Why we remove `virtual` from convert but not from other functions, like loadFrom*?

Reply:
`virtual `is not needed when override is set

Reply:
Then it can be removed for other functions.

Reply:
done